mbtiles/scripts/cli.py

```diff
@@ -94,22 +100,32 @@ def mbtiles(ctx, files, output, force_overwrite, title, description,
                                   force_overwrite=force_overwrite)
     inputfile = files[0]
 
-    logger = logging.getLogger('rio-mbtiles')
+    img_format = img_format.upper()
```

Comment:
This is incredibly minor, but in reading this, I'd find it helpful to move the `img_ext` definition up here with a comment. I went back and forth a couple times looking at why `upper` was being applied here and then `lower` down below.